src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/WebhookMessageSendingQueueProcessor.java

```diff
@@ -76,6 +77,7 @@ private void pollQueue() {
             maybeQueueItem.ifPresent(sendAttempter::attemptSend);
             transaction.commit();
         } catch (Exception e) {
+            Sentry.captureException(e);
             LOGGER.warn("Unexpected exception when polling queue  %s: ".formatted(e.getMessage()));
```

Comment:
If we consider this something that should be reported to Sentry would it be better to log it at an error level and have it propagate through with the dropwizard integration?

Same with the other example here, there may well be reasons to avoid doing that

Reply:
@sfount I agree that is a cleaner approach, have changed it to this suggestion and force-pushed.